### DISCUSSION TITLE
[AI-deck]ERROR with "make autotiler" step when Getting started with the AI deck

### DISCUSSION THREAD

**Discussion by fairwel1** [bitcraze/discussions#1691] [Q&A]
Hi , I was just following the getting started with the AI deck [https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#setup-development-environment](url). And meeting something wrong on Setting up docker and the autotiler with the "make autotiler". The error shows as follows: 
![eeeee](https://github.com/user-attachments/assets/cc7cac44-5ed2-474a-a425-868385b596ae)
Can u give me some help? Thanks!

**Comment by Bananazooka** [comment]
Hello! You marked this as solved, could you tell us how you solved it? It may help others that encounter this issue.

**Reply by fairwel1** [reply]
Maybe u can turn on the VPN and resolving this problem.
